src/screens/DetailGroupImage/index.js

```diff
@@ -74,38 +80,38 @@ export default ShowDetailGroupImage;
 
 const styles = StyleSheet.create({
   image: {
-    width,
+    width: width,
     height: 417,
     marginTop: 8,
     justifyContent: 'flex-end',
     alignItems: 'flex-end',
     paddingBottom: 10,
-    paddingRight: 10
+    paddingRight: 10,
   },
   container: {
-    flex: 1
+    flex: 1,
   },
   header: {
-    backgroundColor: COLORS.holyTosca,
+    backgroundColor: colors.holytosca,
     flexDirection: 'row',
     alignItems: 'center',
     paddingLeft: 20,
     paddingTop: 6,
-    paddingBottom: 7
+    paddingBottom: 7,
   },
   user: {
-    marginLeft: 18
+    marginLeft: 18,
   },
   name: {
     fontFamily: fonts.inter[600],
     fontSize: 14,
-    color: COLORS.white,
-    lineHeight: 16.94
+    color: '#fff',
+    lineHeight: 16.94,
   },
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.white,
-    lineHeight: 18
-  }
+    color: '#fff',
+    lineHeight: 18,
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated to use the new `colors` import. Make sure that the `colors` module contains the `holytosca` property. Also, the color for `name` and `time` has been changed to `#fff`, ensure this change aligns with your design requirements.
```diff
- backgroundColor: COLORS.holyTosca,
- color: COLORS.white,
+ backgroundColor: colors.holytosca,
+ color: '#fff',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->